console/console-init/ui/mock-console-server/mock-console-server.js

```diff
@@ -1804,6 +1804,62 @@ function makeAddressSpaceMetrics(as) {
   ];
 }
 
+function makeIotProjectMetrics(pj) {
+  var devices =
+    iotdevices[getIotDevicesIndexForProjectName(pj.metadata.name)].devices;
+  var cons =
+    as.metadata.uid in addressspace_connection
+      ? addressspace_connection[as.metadata.uid]
+      : [];
+
+  return [
+    {
+      name: "Max connection",
+      type: "counter",
+      value: cons.length,
+      units: "connections"
+    },
+    {
+      name: "Data volume",
+      type: "counter",
+      value: devices.length * 1000,
+      units: "bytes"
+    },
+    {
+      name: "Start date",
+      type: "counter",
+      // value is limited to a float ?
+      value: "11/01/2020",
+      units: "Date"
+    },
+    {
+      name: "End date",
+      type: "counter",
+      // value is limited to a float ?
+      value: "11/01/2021",
+      units: "Date"
```

Comment:
@k-wall any idea on how we could encode a date in there ? The graphQL type is float

Reply:
In the metadata we are exposing timestamps as String! formatted in the ISO-8601 manner, can you reuse the same format.

Sorry I missed this review request.

Reply:
Thanks

Reply:
what I mean is that in the Metric garphQL object the `value` field is typed as a float : 
```
type Metric_consoleapi_enmasse_io_v1beta1 {
  name: String!
  type: MetricType!
  value: Float!
  units: String!
}
```
https://github.com/EnMasseProject/enmasse/blob/2f1c035b1118d825dc66a8ee8e859e5ab0471315/console/console-server/src/main/resources/schema.graphql#L99